tests/test_kernels/test_approximations.py

```diff
@@ -130,12 +130,12 @@ def test_exactness(kernel):
 )
 def test_value_error(kernel):
     with pytest.raises(ValueError):
-        RFF(kernel(), num_basis_fns=10)
+        RFF(base_kernel=kernel(), num_basis_fns=10)
```

Comment:
A better solution would be Python 3.10's KW_ONLY fields for the base classes

Reply:
I would love to depend on 3.10. I worry that it's too restrictive though. Maybe this warrants more discussion.